examples/client/Locomotion/src/Components/CardRow/index.tsx

```diff
@@ -126,8 +105,16 @@ const CardRow = (paymentMethod: any) => (
               {isCashPaymentMethod(paymentMethod)
                 ? <SvgIcon Svg={cashIcon} width={40} height={25} />
                 : <PaymentIcon type={paymentMethod.brand} />}
-              {paymentMethod.mark ? (isCashPaymentMethod(paymentMethod)
-                ? CashSelected : CreditCardSelected) : null }
+              {paymentMethod.mark ? (
+                <SvgIcon
+                  style={{
+                    position: 'absolute',
+                    right: -7,
+                    bottom: -7,
+                  }}
+                  Svg={selected}
+                />
+              ) : null }
```

Comment:
did you check cash and card?  cash svg was positioned else

Reply:
Yes, both on android and ios